src/Storage/FileSystem/Physical/PhysicalFile.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
+
+namespace DotNetRu.Auditor.Storage.FileSystem.Physical
+{
+    public sealed class PhysicalFile : FileSystemEntry, IFile
+    {
+        public PhysicalFile(string fullName)
+            : base(fullName, File.Exists(fullName))
+        {
+        }
+
+        public Stream OpenForRead()
+        {
+            if (!Exists)
+            {
+                throw NotFoundFile.ToException(FullName);
+            }
+
+            return File.OpenRead(FullName);
+        }
+
+        public Stream OpenForWrite()
+        {
+            if (!Exists)
+            {
+                throw NotFoundFile.ToException(FullName);
+            }
+
+            return File.OpenWrite(FullName);

```

Comment:
```suggestion
            return new ValueTask<Stream>(File.OpenWrite(FullName));
```
Возможно, это не самый правильный способ создать `ValueTask<>`, но я не знаю про другие

Reply:
Обычно делают:
`ValueTask.FromResult(x)`
Это позволяет выводить дженерик параметр. Но в данном случае всё едино.